planemo/workflow_lint.py

```diff
@@ -233,18 +233,16 @@ def _check_test_assertions(lint_context, assertion_definitions):
     # Python functions directly, rather than checking against galaxy.xsd as for tool linting
     assertions_valid = True
     if assertion_definitions:
-        for module in asserts.assertion_modules:
-            for function_name in dir(module):
-                if function_name.split("assert_")[-1] in assertion_definitions:
-                    signature = inspect.signature(module.__dict__[function_name])
-                    try:
-                        # try mapping the function with the attributes supplied and check for TypeError
-                        signature.bind("", **assertion_definitions[function_name.split("assert_")[-1]])
-                    except AssertionError:
-                        pass
-                    except TypeError as e:
-                        lint_context.error(f"Invalid assertion in tests: {function_name} {str(e)}")
-                        assertions_valid = False
+        for function_name, function in asserts.assertion_functions.items():
```

Comment:
I think that means we also need to pin a minimum galaxy-tool-util version (or make this compatible with both)

Reply:
I see. So I just require `>=22.01` for all (or is is `>=22.1`?

Reply:
It's 22.1